app/assets/stylesheets/active_admin/pages/_logged_out.scss

```diff
@@ -1,44 +1,30 @@
-body.logged_out {
-  background: #e8e9ea;
-
-  #content_wrapper{
-    width: 500px;
-    margin: 70px auto;
-    #active_admin_content {
-      @include shadow;
-      background: #fff;
-      padding: 13px 30px;
-    }
-  }
+#login {
+  background-color: $panel-background-color;
+  border-radius: $border-radius-base;
+  margin: 70px auto;
+  padding: 15px;
+  width: 500px;
 
   h2 {
-    @include section-header;
-    @include primary-gradient;
-    @include text-shadow(#000);
-    box-shadow: 0 1px 3px rgba(0,0,0,0.3);
-    border: none;
-    color: #fff;
-    margin: -13px -30px 20px -30px;
+    font-size: 1.8em;
   }
 
-  #login {
-    /* Login Form */
-    form {
-      fieldset {
-        @include no-shadow;
-        background: none;
-        padding: 0;
-        li { padding: 10px 0; }
+  fieldset.inputs {
+    background-color: transparent;
+    margin-bottom: 0;
+    padding: 0;
 
-        input[type=text], input[type=email], input[type=password] {
-          width: 70%;
-        }
-        &.buttons { margin-left: 20%; }
-        margin-bottom: 0;
-      }
+    input[type=text],
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3